sematic/storage.py

```diff
@@ -94,27 +139,39 @@ def _bucket(self) -> str:
     def _s3_client(self):
         return boto3.client("s3")
 
+    def _make_presigned_url(self, client_method: S3ClientMethod, key: str) -> str:
+        presigned_url = self._s3_client.generate_presigned_url(
+            ClientMethod=client_method.value,
+            Params={"Bucket": self._bucket, "Key": key},
+            ExpiresIn=5 * 60,
```

Comment:
Can we move this constant to the top of the file?

Reply:
Done.